pkg/zcmd/zfs/get/builder.go

```diff
@@ -169,37 +168,37 @@ func (v *VolumeGetProperty) Build() (*VolumeGetProperty, error) {
 	v = v.Validate()
```

Comment:
Here, When the caller makes a call to `v.Build()` it will return CLI command based on the options... for example if the caller needs information about volume it will return the command as
`zfs get -H -p -o value,property, used,available <volume_name>` but with this changes now it will return `zfsget -H -p -o propertyvalue usedavailable <volume_name>`